### PR TITLE
antu2-16-11-2024

### PR DESCRIPTION
**Pull request by singhantu**

### CODE REVIEW

File: BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js [hunk 1]
```diff
@@ -14,9 +14,11 @@ function GetApprovedQuotationList() {
         const ShowButton = ["V"];
```
Comment (erpbizsol) [inline BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js:14]: hi

Comment (singhantu) [reply BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js:14]: me

File: BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js [hunk 2]
```diff
@@ -14,9 +14,11 @@ function GetApprovedQuotationList() {
...
         const hiddenColumns = ["Code"];
         const showButtons = [];
-        const updatedResponse = resData.map(item => ({...item, Action: item.Action ? `<button style="background-color:#198754;border-radius: 5px; " onclick="ViewData('${item.Code}')"><i class="fa-solid fa-folder-open" style="color:white;"></i></button>
+        const updatedResponse = resData.map(item => ({
```
Comment (erpbizsol) [inline BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js:17]: why thi

Comment (singhantu) [reply BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js:17]: no

Comment (singhantu) [reply BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js:17]: nu

Comment (erpbizsol) [reply BizsolERP/Bizsol.ERP.UI.Purchase.Transactions/wwwroot/Script/QuotationApproval.js:17]: plz see code attachment
Screenshot 2023-05-06 103104

**Review by erpbizsol — APPROVED** [review]

**Pull request merged**